### DISCUSSION TITLE
[@types/node]: globals.d.ts:72:13 - error TS2403 with TypeScript 4.9.3

### DISCUSSION THREAD

**Discussion by IvanPizhenko** [DefinitelyTyped/DefinitelyTyped#63270] [Issues with a @types package]
After upgrading TypeScript from 4.8.4 to 4.9.3:
```
node_modules/@types/node/globals.d.ts:72:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'AbortSignal' must be of type '{ new (): AbortSignal; prototype: AbortSignal; abort(reason?: any): AbortSignal; timeout(milliseconds: number): AbortSignal; }', but here has type '{ new (): AbortSignal; prototype: AbortSignal; }'.

72 declare var AbortSignal: {
               ~~~~~~~~~~~

  node_modules/typescript/lib/lib.dom.d.ts:2071:13
    2071 declare var AbortSignal: {
                     ~~~~~~~~~~~
    'AbortSignal' was also declared here.


Found 1 error in node_modules/@types/node/globals.d.ts:72
```

[a comment by an automated account is not shown]

**Comment by IvanPizhenko** [comment]
https://github.com/microsoft, https://github.com/DefinitelyTyped, @jkomyno, @alvis, @r3nya, @btoueg, @smac89, @Touffy, @DeividasBakanas, @eyqs, @Hannes-Magnusson-CK, @hoo29, @kjin, @ajafff, @islishude, @mwiktorczyk, @mohsen1, @n-e, @galkin, @parambirs, @eps1lon, @SimonSchick, @ThomasdenH, @WilcoBakker, @wwwy3y3, @samuela, @kuehlein, @bhongy, @chyzwar, @trivikr, @yoursunny, @qwelias, @ExE-Boss, @peterblazejewicz, @addaleax, @victorperin, @ZYSzys, https://github.com/nodejs, @LinusU, @wafuwafu13, @mcollina. 

does anyone can fix this?

**Reply by peterblazejewicz** [reply]
please update your @types/node installation. There is a TS version-dependent implementation now,
similar: #63272
